python/CMakeLists.txt

```diff
@@ -1,14 +1,19 @@
-find_package(pybind11 2.2 CONFIG)
+find_package(pybind11 2.11 CONFIG)
 
 if(NOT pybind11_FOUND)
   add_subdirectory( pybind11 )
   # SYSTEM is supported for embedded pybind source (>= 2.3.0)
-  set(PYBIND11_SYSTEM "SYSTEM")
+  # pybind11_add_module SYSTEM argument was deprecated in pybind11 version 2.6
+  # See: https://pybind11.readthedocs.io/en/stable/upgrade.html
+  #  if(pybind11_VERSION VERSION_LESS "2.6.0")
+  #      set(PYBIND11_SYSTEM "SYSTEM")
```

Comment:
Commented out `set(PYBIND11_SYSTEM "SYSTEM")` here in accordance with https://pybind11.readthedocs.io/en/stable/upgrade.html. The upgrade notes for pybind11 2.6.0 reads:

> pybind11_add_module’s SYSTEM argument is deprecated and does nothing; linking now behaves like > other imported libraries consistently in both config and submodule mode, and behaves like a SYSTEM > library by default.